### ISSUE TITLE
Can anyone make the Megamix mod but delete every other week except week 1-7

### ISSUE THREAD

**Issue by kffwlp** [luckydog7/Funkin-android#916]
Can anyone make #817 without the mod weeks (whitty, tricky, etc.) Or if you don't want to please teach me how to delete them.

**Comment by itzmedavidmodder** [comment]
Yeah bro i have It this is the link:https://www.mediafire.com/file/gulkmng2wm3q7rb/Friday_night_funkin_Lite_by_Lucas_Adv_b%2528Fnf_Tik_Tok_engine%2529.apk/file
Creduta:Electry Zack (on YouTube)

**Comment by kffwlp** [comment]
> Yeah bro i have It this is the link:https://www.mediafire.com/file/gulkmng2wm3q7rb/Friday_night_funkin_Lite_by_Lucas_Adv_b%2528Fnf_Tik_Tok_engine%2529.apk/file
> Creduta:Electry Zack (on YouTube)

Thanks a lot! 😄 It runs more smoothly now

**Comment by itzmedavidmodder** [comment]
You're welcome bro

**Comment by itzmedavidmodder** [comment]
Ya bro It runa smoothly because this is optimized version:this version has best sprite qualità,RAM optimization,and smooth running with 90 fps

**Comment by itzmedavidmodder** [comment]
Ops.. Sorry bro if i make mistakes when i type,because in italian and i have keyboard language setted on italian

**Comment by BoyfriendFNF** [comment]
> Yeah bro i have It this is the link:https://www.mediafire.com/file/gulkmng2wm3q7rb/Friday_night_funkin_Lite_by_Lucas_Adv_b%2528Fnf_Tik_Tok_engine%2529.apk/file
> Creduta:Electry Zack (on YouTube)

it's missing

**Comment by DabKnight** [comment]
> > Yeah bro i have It this is the link:https://www.mediafire.com/file/gulkmng2wm3q7rb/Friday_night_funkin_Lite_by_Lucas_Adv_b%2528Fnf_Tik_Tok_engine%2529.apk/file
> > Creduta:Electry Zack (on YouTube)
> 
> Thanks a lot! 😄 It runs more smoothly now

Dude,its gone.
Did you delete it or something?
